benchmark/flexsn/benchmark_vgg_inductor.py

```
@@ -0,0 +1,146 @@
+    x = torch.randn(T, B, C, H, W, device="cuda")
+
+    configs = [
+        ("LIFNode  backend=torch  ",
+         lambda: build_vgg(neuron.LIFNode, backend="torch")),
+        ("LIFNode  backend=triton ",
+         lambda: build_vgg(neuron.LIFNode, backend="triton")),
+        ("FlexSN   backend=inductor",
+         lambda: build_vgg(make_flexsn_factory())),
+    ]
+
+    print(f"Input : T={T}, B={B}, {C}×{H}×{W}  (ImageNet-scale)")
```

Ruff (0.15.11)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)